src/bindings.d.ts

```diff
@@ -15,4 +15,5 @@ export interface Environment
   CONTENT_CLAIMS_SERVICE_URL?: string
   HONEYCOMB_API_KEY: string
   FF_TELEMETRY_ENABLED: string
+  FF_TELEMETRY_RATIO: string
```

Comment:
Not a feature flag, but optional config.

```suggestion
  TELEMETRY_RATIO: string
```